DigitalLearningSolutions.Web/Helpers/RegisterAdminHelper.cs

```diff
@@ -21,9 +25,11 @@ ICentresDataService centresDataService
         public bool IsRegisterAdminAllowed(int centreId)
         {
             var admins = userDataService.GetAdminsByCentreId(centreId);
+            var centre = centresDataService.GetCentreDetailsById(centreId);
             var hasCentreManagerAdmin = admins.Any(admin => admin.AdminAccount.IsCentreManager);
             var (autoRegistered, autoRegisterManagerEmail) = centresDataService.GetCentreAutoRegisterValues(centreId);
-            return !hasCentreManagerAdmin && !autoRegistered && !string.IsNullOrWhiteSpace(autoRegisterManagerEmail);
+            return centre!.Active && !hasCentreManagerAdmin && !autoRegistered &&
```

Comment:
I think this should be `centre?.Active` instead of `centre!.Active`, since there is no guarantee within this method that the input centre ID exists for a centre in the database